app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/table.tsx

```diff
@@ -215,21 +266,32 @@ export const renderTable = ({
                   </CenteredTextSpan>
                 </GridCell>
                 <GridCell item xs={6}>
-                  <CenteredTextSpan>{isPrimaryKey ? 'Primary' : '--'}</CenteredTextSpan>
+                  <PrimaryKeySpan>{isPrimaryKey ? 'Primary' : '--'}</PrimaryKeySpan>
                 </GridCell>
               </GridDividerCell>
               <GridCellContainer item xs={4} container direction="row">
                 <GridButtonCell item xs={2}>
-                  <StatusButton status="success" />
+                  {Boolean(assessmentForCol) && (
```

Comment:
Nit: Can we do `!!foo` instead of `Boolean(foo)`?

Reply:
hmm I guess I've been seeing Boolean used more in docs etc so I started using it - https://google.github.io/styleguide/tsguide.html#type-coercion says that either is fine for type coercion. I think it makes sense to just keep using `!!` and can flip them all back to that